src/BuildingBlocks/EventBus/EventBus/Abstractions/IEventBus.cs

```diff
@@ -19,5 +19,7 @@ void UnsubscribeDynamic<TH>(string eventName)
         void Unsubscribe<T, TH>()
             where TH : IIntegrationEventHandler<T>
             where T : IntegrationEvent;
+
+        void Dispose();
```

Comment:
rather than modify this interface, we should just have EventBusServiceBus implement IDisposable, similar to https://github.com/dotnet-architecture/eShopOnContainers/blob/19baeb70699161b3e964e583ef43b444a949cf19/src/BuildingBlocks/EventBus/EventBusRabbitMQ/EventBusRabbitMQ.cs#L20